build.gradle.kts

```diff
@@ -44,29 +47,36 @@ sourceSets {
     }
 }
 
-val integrationTestImplementation by configurations.getting { extendsFrom(configurations.implementation.get()) }
-val integrationTestRuntimeOnly: Configuration by configurations.getting
+val integrationTestSourceSet: SourceSet = sourceSets["integrationTest"]
 
-configurations["integrationTestRuntimeOnly"].extendsFrom(configurations.runtimeOnly.get())
+val integrationTestImplementation: Configuration by
+    configurations.getting { extendsFrom(configurations.implementation.get()) }
+val integrationTestRuntimeOnly: Configuration by
+    configurations.getting { extendsFrom(configurations.runtimeOnly.get()) }
 
-val integrationTest =
+val integrationTestTask: Task =
     task<Test>("integrationTest") {
-        description = "Runs integration tests requiring external MongoDB deployment"
         group = "verification"
 
-        testClassesDirs = sourceSets["integrationTest"].output.classesDirs
-        classpath = sourceSets["integrationTest"].runtimeClasspath
-        shouldRunAfter("test")
+        testClassesDirs = integrationTestSourceSet.output.classesDirs
+        classpath = integrationTestSourceSet.runtimeClasspath
 
         useJUnitPlatform()
-
-        testLogging { events("passed") }
```

Comment:
the above comes from gradle doc. I doubt we need it for one thing it lacks "skipped" and "failed"; for another thing, if the `test` task doens't specify it, why not do the same thing for `integration test`?